apps/fyle/signals.py

```
@@ -0,0 +1,25 @@
+
+@receiver(post_save, sender=ExpenseFilter)
+def run_post_save_expense_filters(sender, instance: ExpenseFilter, **kwargs):
+    """
+    :param sender: Sender Class
+    :param instance: Row Instance of Sender Class
+    :return: None
+    """
+    try:
+        run_post_save_expense_filters(instance)
+    except Exception as e:
+        raise ValidationError(e)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)